scenes/src/app/scenes/badge/badge-scene.scss

```diff
@@ -0,0 +1,4 @@
+.scene-badge-icon {
+  color: #333333;
```

Comment:
nit: You can use the 3 character shortcut `#333` (this shortcut works when each color channel is just the same digit twice - e.g. `#abc === #aabbcc`)